usermods/Internal_Temperature_v2/usermod_internal_temperature.h

```diff
@@ -0,0 +1,134 @@
+#pragma once
+
+#include "wled.h"
+
+class InternalTemperatureUsermod : public Usermod
+{
+
+private:
+  unsigned long loopInterval;
+  unsigned long lastTime = 0;
+  bool isEnabled;
```

Comment:
Should have a default value. Maybe start "disabled" by default, as it's possible that the undocumented temperature read function has unexpected side-effects.

Reply:
It has a default value, see line 98.